### ISSUE TITLE
Empty cell in table error

### ISSUE THREAD

**Issue by SurglogsGithubUser** [Tyler-Keith-Thompson/CucumberSwift#86]
**Describe the bug**
When there is no value in table, there is error mesages (and also it will fail tests).
I think there is error with symbols < > (as mentioned [here](https://github.com/Tyler-Keith-Thompson/CucumberSwift/issues/82)) and also with empty values
Error messages: 

```
XCTAssertTrue failed - Gherkin language errors found:
File: entry.feature, table header <none> not found
File: entry.feature, table header <none> not found
File: entry.feature, table header <none> not found
File: entry.feature, table header <none> not found
File: entry.feature, table header <none> not found
File: entry.feature, table header <none> not found
File: entry.feature, table header <none> not found
File: entry.feature, table header <none> not found
File: entry.feature, table header <none> not found
File: entry.feature, table header <none> not found
File: entry.feature, table header <none> not found
File: entry.feature, table header <none> not found
```

**Additional context**
 - XCode Version: XCode 14.2
 - CucumberSwift Version: 4.2.1
 - Example Feature File:
 
  @c1178344 @alternative @smartphone @test
  Scenario: View ignored entry for archived log on smartphone

```
    When  User clicks on "Temperature and Humidity Log" log
    And   User clicks on 'filter' menu item
    And   User clicks on "Show ignored" button
    And   User clicks on "Close" button
    Then  User should see log entries:
      | TYPE    | NAME       | SUMMARY              | LABEL     | ACTIONS |
      | missed  | 01/06/2021 | Missed check         |           | <none>  |
      | missed  | 01/05/2021 | Missed check         |           | <none>  |
      | checked | 01/04/2021 | Logged by Test user  |           | <none>  |
      | checked | 01/03/2021 | Logged by Test user  |           | <none>  |
      | ignored | 01/02/2021 | Ignored by Test user | (Ignored) | <none>  |
      | ignored | 01/01/2021 | Ignored by Test user | (Ignored) | <none>  |
    When  User clicks on "01/01/2021" entry
    Then  User should see "SL - Log entry" screen
    And   User should see "12:00 AM on Friday, January 01, 2021" text
    And   User should see "Logged by Test user" text
    And   User should see "Show Edit History" button
    And   User should see "Reason/Note" text
    And   User should see "Humidity indicator failure." text
    But   User should not see 'edit' menu item

```

**Comment by Tyler-Keith-Thompson** [comment]
Just to clarify there's 2 problems here. Once is that LABEL has empty values which aren't behaving as expected and additionally the ACTIONS of `<none>` are also erroring out?

I think I understand why this would happen. I'll take a look as soon as I can, but PRs are also welcome.

**Comment by SurglogsGithubUser** [comment]
> Just to clarify there's 2 problems here. Once is that LABEL has empty values which aren't behaving as expected and additionally the ACTIONS of `<none>` are also erroring out?
> 
> I think I understand why this would happen. I'll take a look as soon as I can, but PRs are also welcome.

Correct

**Comment by Tyler-Keith-Thompson** [comment]
This should be resolved in CucumberSwift v4.2.2
